src/tableDrawer.ts

```diff
@@ -34,19 +35,73 @@ export function drawTable(jsPDFDoc: jsPDFDocument, table: Table): void {
   const startPos = assign({}, cursor)
 
   table.startPageNumber = doc.pageNumber()
+  console.log('inside splitColumns - 0 ****** ', settings)
 
-  doc.applyStyles(doc.userStyles)
-  if (settings.showHead === 'firstPage' || settings.showHead === 'everyPage') {
-    table.head.forEach((row) => printRow(doc, table, row, cursor))
-  }
-  doc.applyStyles(doc.userStyles)
-  table.body.forEach((row, index) => {
-    const isLastRow = index === table.body.length - 1
-    printFullRow(doc, table, row, isLastRow, startPos, cursor)
-  })
-  doc.applyStyles(doc.userStyles)
-  if (settings.showFoot === 'lastPage' || settings.showFoot === 'everyPage') {
-    table.foot.forEach((row) => printRow(doc, table, row, cursor))
+  if (settings.splitColumns === true) {
```

Comment:
We should probably split up drawTable to make it less complex. Trying to keep methods to about 50 lines so they can be overviewed easily.